src/css/components/_captions-settings.scss

```diff
@@ -41,21 +41,36 @@
   }
 
   .vjs-text-track-settings .vjs-track-settings-controls {
-    grid-column: 2;
+    grid-column: 1 / -1;
     grid-row: 2;
+  }
+
+
+  // 1 column for small players
+  @media (max-width: 425px) {
```

Comment:
what about using the responsive classes instead of media queries?
Also, shouldn't grid automatically resize if we set a minimum size?

Reply:
If responsive was the default, I would think that would be a good idea. Going to check on the grid with a minimum size.

Reply:
I think it's reasonable to use that feature, though. The idea of the feature is that components can use it to be able to expect what size the player is without doing extra stuff in javascript themselves. I'd want to have it enabled by default eventually.

But if it can just automatically resize based on grid, that would be even better, probably.

Reply:
Seems like we cannot auto-resize the grid, at least from all I have tried, I changed it to use the responsive class.